### ISSUE TITLE
False positive FreeLetInFunc in presence of `inherit`

### ISSUE THREAD

**Issue by mcwitt** [Synthetica9/nix-linter#64]
In version `0.2.0.3` (revision `38c4a14681cf3a1e6f098d8b723db503910a28d8`), the input

```nix
foo:
let inherit (foo) bar;
in bar
```
results in

```
Move `let` block outside function definition at foo.nix:1:1-4:1
```

but the `let` block can't be moved outside of the function definition since it depends on the argument `foo`.

Possibly related to #61, #62 ?
